docs/key.md

```diff
@@ -18,11 +18,15 @@ There are 3 permission settings for a key:
 * both (read and write)
 
 To create a key with a desired permission, run:
-`google-cloud-gemserver create_key --permissions [read/write/both]`, i.e. `google-cloud-gemserver create_key permissions write`. By default, if the permissions argument is not given a key will be generated
-with both (read, write) permissions.
+`google-cloud-gemserver create-key --permissions [read/write/both] --use-proj PROJECT_ID`, i.e. `google-cloud-gemserver create-key permissions write --use-proj PROJECT_ID`. By default, if the permissions argument is not given a key will be generated with both (read, write) permissions.
```

Comment:
Use `read|write|both` which should be common enough that you do not need the "i.e." section. (If you do keep that section, "e.g." is more correct than "i.e." for providing an example, and you're missing the "--" before permissions.)